src/common/components/modal/Modal.js

```diff
@@ -78,10 +86,10 @@ function Modal({ children, onModalCloseClick }) {
             role="button"
             tabIndex={0}
           >
-            X
+            <img src={iconClose} alt="닫기" />
           </div>
         </header>
-        {children}
+        <body>{children}</body>
```

Comment:
저는 html 문서내에 body 태그는 1개만 존재 할 수 있는걸로 아는데요. 혹시 정말 특별한 이유가 있는것이 아니라면 body로 감싸지 않는게 좋지 않을까요?

https://www.w3schools.com/tags/tag_body.asp

```suggestion
        {children}
```

Reply:
헐.... 새벽 무지성 코딩의 결과네요;;; 수정수정... ㅠ